examples/v1beta1/al2-kubelet-log-query.yaml

```diff
@@ -0,0 +1,49 @@
+# This example NodePool will provision instances using the AL2 EKS-Optimized AMI
```

Comment:
(nit) When a YAML file has a `---` line, I put one at the start as well.

That helps people learn that `---` is a document start marker, not a separator.